test/functional/activities_controller_test.rb

```diff
@@ -77,7 +79,7 @@ def test_project_index
                }
   end
 
-  def test_previous_project_index
+  it 'previous_project_index' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.